siliconcompiler/tools/openroad/openroad.py

```diff
@@ -438,27 +436,21 @@ def get_corner_by_check(chip, check):
                                  step=step, index=index):
             continue
 
-        libcorner = chip.get('constraint', 'timing', constraint, 'libcorner',
-                             step=step, index=index)
-        if libcorner:
-            return libcorner[0]
+        return constraint
```

Comment:
This can now be rewritten to:
```suggestion
    if check in chip.get('constraint', 'timing', constraint, 'check',
                                 step=step, index=index):
        return constraint
```